### ISSUE TITLE
Module loading did not work.

### ISSUE THREAD

**Issue by coloredshark** [s0lst1c3/dropengine#1]
This may be because I am using python 3.8.5  instead of 3.7 (not for lack of trying as I was unable to get a virtual environment setup with any earlier versions of python) but i keep getting the following error when i try to run dropengine.py with any or no arguments 
```
Crypter modules requires attribute: self.compatible_decrypter
ERROR:root:Module loading did not work
Crypter modules requires attribute: self.compatible_decrypter
ERROR:root:Module loading did not work
[!] Error: module MDKey must have attribute: self.compatible_ekeys
```
perhaps you could point me in the right direction as to what this error is from or verify if the python version is the issue or not.
I really am eager to try the tool out.  It looked pretty interesting in the defcon presentation.
Thanks in advanced.

**Comment by s0lst1c3** [comment]
Hi @coloredshark,

I actually developed DropEngine using Python 3.8.2, which shouldn't be that dissimilar to what you're using. Can you please provide me with the exact command you ran in order to produce this error?

**Comment by coloredshark** [comment]
Any command I try to run produces that output.  Output specifically is from running it without any arguments.
```
> python dropengine.py

Crypter modules requires attribute: self.compatible_decrypter
ERROR:root:Module loading did not work
Crypter modules requires attribute: self.compatible_decrypter
ERROR:root:Module loading did not work
[!] Error: module MDKey must have attribute: self.compatible_ekeys
```

I have tried other commands and everything produces the same output regardless of arguments provided.

I have actually tried it on a few different machines, with and without attempting to make virtual environments and the result has been the same every time.  I attempted to run it on 1 machine running Ubuntu and 2 running Debian,  all were python version 3.8.5.  I did attempt to install 3.7 in venv using the python3.7-venv package as well as the python3-venv packages but all environments I set up always gave the same 3.8.5 result when querying the version.  Hopefully that information is helpful in some way.  I started to attempted to look at your code myself but I already have projects going on myself and your tool looked so nice I was hoping to save myself some time with it so I decided against trying to figure it out myself lol.  If you want any other information let me know.
